### DISCUSSION TITLE
keine Verbindung von raspberrymatic über homematic local auf homeassistant

### DISCUSSION THREAD

**Discussion by tompanj** [OpenCCU/OpenCCU#2115] [🇩🇪 Fragen&Antworten]
### Describe the issue you are experiencing

Habe raspberrymatic auf homeassistant installiert, die Einbindung der Geräte funktionierte einwandfrei. Danach wollte ich mit der homematic local die Verbindung herstellen zwischen raspberrymatic und homeassistant. Die Eingabe von hostnamen, benutzer ( admin) und Passwort sind korrekt. Es kommt eine Fehlermeldung Verbindung nicht möglich und er findet die Geräte nicht. Anbei noch einige Screenshots zur besseren Verständnis. Wer kann eventuell helfen. Bei noch fehlenden angaben zur Lösung bitte schreiben.

### Describe the behavior you expected

Integration von rasberrymatic in den homeassistant

### Steps to reproduce the issue

1.keine angaben
2.
3.
...

### What is the version this bug report is based on?

3.65.11.20221218

### Which base platform are you running?

rpi4 (RaspberryPi4)

### Which HomeMatic/homematicIP radio module are you using?

HmIP-RFUSB

### Anything in the logs that might be useful for us?

```txt
keine Angaben
```

**Comment by jens-maus** [comment]
Da dies kein Bug/Problem in RaspberryMatic selbst ist, sondern ein Nutzungsproblem habe ich dies mal in einen Diskussionsbeitrag konvertiert. Des Weiteren habe ich die Word-Dateien entfernt weil sich so etwas nicht gehört das man screenshots in Word-Dateien einbettet. Da kann man gleich das ganze noch ausdrucken und dann wieder einscannen um es hier zu präsentieren :)
